indico/modules/categories/blueprint.py

```diff
@@ -26,6 +27,10 @@
 
 # Management
 _bp.add_url_rule('/manage/', 'manage_content', RHManageCategoryContent)
+_bp.add_url_rule('/manage/settings/', 'manage_settings', RHManageCategorySettings, methods=('POST', 'GET'))
+_bp.add_url_rule('/manage/settings/upload_icon', 'upload_icon', RHSettingsIconUpload, methods=('POST',))
```

Comment:
Wouldn't it make more sense to use a single url/RH for uploading/deleting an icon?

    _bp.add_url_rule('/manage/icon', 'upload_icon', RHManageCategoryIcon, methods=('POST', 'DELETE'))

Since we have no standalone form it seems like a perfect case for something a bit more RESTful